.github/workflows/runtests.yml

```diff
@@ -11,15 +11,15 @@ on:
 jobs:
   test:
     name: Julia ${{ matrix.version }} - ${{ matrix.tests }} - ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
+    runs-on: self-hosted
     env:
       test_suite:  ${{ matrix.tests }}
 
     strategy:
       fail-fast: false
 
       matrix:
-        version: ['1.6', '1.7', '1.8']
+        version: ['1.6', '1.7', '1.8', '1.10']
```

Comment:
1.9?

Reply:
Wanted to start with the superset of 1.10, and then add 1.9. In retrospect might have been smarter to go more incrementally and do 1.9 first. Will add 1.9 in a bit

Reply:
Oh no I meant adding both 😆